### PR TITLE
changes

### PR DESCRIPTION
**Pull request by isahwahni**
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/ibatta/SelFW/pull/6%23discussion_r151412226%22%2C%20%22https%3A//github.com/ibatta/SelFW/pull/6%23discussion_r151414280%22%2C%20%22https%3A//github.com/ibatta/SelFW/pull/6%23discussion_r151414573%22%2C%20%22https%3A//github.com/ibatta/SelFW/pull/6%23discussion_r151414576%22%5D%2C%20%22comments%22%3A%20%7B%22Pull%20e8bfbcb28fba1b91f0047cdc442fbeb0bd259ac6%20src/com/generic/reports/reportsTemplates/failedTCTemplate.html%207%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/ibatta/SelFW/pull/6%23discussion_r151414280%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22revert%20to%20initial%20values%20%22%2C%20%22created_at%22%3A%20%222017-11-16T13%3A36%3A58Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars2.githubusercontent.com/u/18301319%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/ibatta%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%3Ca%20href%3D%27%23crh-update-none%27%3E%3C/a%3E%22%2C%20%22created_at%22%3A%20%222017-11-16T13%3A38%3A19Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars3.githubusercontent.com/u/33652900%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/isahwahni%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20src/com/generic/reports/reportsTemplates/failedTCTemplate.html%3AL12-20%22%7D%2C%20%22Pull%20ee243a7e9e7e06fbee1b238a1afaeb4c197435f3%20src/com/generic/reports/reportsTemplates/failedTCTemplate.html%207%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/ibatta/SelFW/pull/6%23discussion_r151412226%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%40isahwahni%20please%20return%20this%20value%20back%20to%2014%25%20%5Cr%5Cn%22%2C%20%22created_at%22%3A%20%222017-11-16T13%3A27%3A32Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars2.githubusercontent.com/u/18301319%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/ibatta%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%3Ca%20href%3D%27%23crh-update-complete%27%3E%3C/a%3E%22%2C%20%22created_at%22%3A%20%222017-11-16T13%3A38%3A20Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars3.githubusercontent.com/u/33652900%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/isahwahni%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20src/com/generic/reports/reportsTemplates/failedTCTemplate.html%3AL12-20%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [x] <a href='#crh-comment-Pull ee243a7e9e7e06fbee1b238a1afaeb4c197435f3 src/com/generic/reports/reportsTemplates/failedTCTemplate.html 7'></a> &nbsp;<b><a href='https://github.com/ibatta/SelFW/pull/6#discussion_r151412226'>File: src/com/generic/reports/reportsTemplates/failedTCTemplate.html:L12-20</a></b>
- <a href='https://github.com/ibatta'></a> @isahwahni please return this value back to 14%
- [x] <a href='#crh-comment-Pull e8bfbcb28fba1b91f0047cdc442fbeb0bd259ac6 src/com/generic/reports/reportsTemplates/failedTCTemplate.html 7'></a> &nbsp;<b><a href='https://github.com/ibatta/SelFW/pull/6#discussion_r151414280'>File: src/com/generic/reports/reportsTemplates/failedTCTemplate.html:L12-20</a></b>
- <a href='https://github.com/ibatta'></a> revert to initial values

<a href='https://www.codereviewhub.com/ibatta/SelFW/pull/6?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/ibatta/SelFW/pull/6?approve=1'></a>&nbsp;<a href='https://github.com/ibatta/SelFW/pull/6'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by ibatta** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/ibatta/SelFW/pull/6'>Refresh</a>.<a href='#crh-update-none'></a>

File: src/com/generic/reports/reportsTemplates/failedTCTemplate.html [hunk 1]
```diff
@@ -12,8 +12,8 @@
 </tr>
 <tr>
 <td align=center width=10%><FONT COLOR=#153E7E FACE=Arial SIZE=2><b>~TSNum~</b></td>
-<td width=75% align= center bgcolor=Red><FONT COLOR=#153E7E FACE=Arial SIZE=2><b>~Fail Desc~</b></td>
-<td align=center width=15%><FONT COLOR=#153E7E FACE=Arial SIZE=2><b><a href=~ScreenshotPath~ target=_blank>Screen Shot</a></b></td>
+<td width=70% align= center bgcolor=Red><FONT COLOR=#153E7E FACE=Arial SIZE=2><b>~Fail Desc~</b></td>
+<td align=center width=20%><FONT COLOR=#153E7E FACE=Arial SIZE=2><b><a href=~ScreenshotPath~ target=_blank>Screen Shot</a></b></td>
```
Comment (ibatta) [inline src/com/generic/reports/reportsTemplates/failedTCTemplate.html]: @isahwahni please return this value back to 14%

File: src/com/generic/reports/reportsTemplates/failedTCTemplate.html [hunk 2]
```diff
@@ -12,8 +12,8 @@
 </tr>
 <tr>
 <td align=center width=10%><FONT COLOR=#153E7E FACE=Arial SIZE=2><b>~TSNum~</b></td>
-<td width=70% align= center bgcolor=Red><FONT COLOR=#153E7E FACE=Arial SIZE=2><b>~Fail Desc~</b></td>
-<td align=center width=20%><FONT COLOR=#153E7E FACE=Arial SIZE=2><b><a href=~ScreenshotPath~ target=_blank>Screen Shot</a></b></td>
+<td width=76% align= center bgcolor=Red><FONT COLOR=#153E7E FACE=Arial SIZE=2><b>~Fail Desc~</b></td>
+<td align=center width=14%><FONT COLOR=#153E7E FACE=Arial SIZE=2><b><a href=~ScreenshotPath~ target=_blank>Screen Shot</a></b></td>
```
Comment (ibatta) [inline src/com/generic/reports/reportsTemplates/failedTCTemplate.html]: revert to initial values

File: src/com/generic/reports/reportsTemplates/failedTCTemplate.html [hunk 3]
```diff
@@ -12,8 +12,8 @@
 </tr>
 <tr>
 <td align=center width=10%><FONT COLOR=#153E7E FACE=Arial SIZE=2><b>~TSNum~</b></td>
-<td width=75% align= center bgcolor=Red><FONT COLOR=#153E7E FACE=Arial SIZE=2><b>~Fail Desc~</b></td>
-<td align=center width=15%><FONT COLOR=#153E7E FACE=Arial SIZE=2><b><a href=~ScreenshotPath~ target=_blank>Screen Shot</a></b></td>
+<td width=76% align= center bgcolor=Red><FONT COLOR=#153E7E FACE=Arial SIZE=2><b>~Fail Desc~</b></td>
+<td align=center width=14%><FONT COLOR=#153E7E FACE=Arial SIZE=2><b><a href=~ScreenshotPath~ target=_blank>Screen Shot</a></b></td>
```
Comment (isahwahni) [inline src/com/generic/reports/reportsTemplates/failedTCTemplate.html]: :+1:<a href='#crh-update-none'></a>

File: src/com/generic/reports/reportsTemplates/failedTCTemplate.html (hunk 1 above)
Comment (isahwahni) [inline src/com/generic/reports/reportsTemplates/failedTCTemplate.html]: :+1:<a href='#crh-update-complete'></a>

**Pull request merged**